Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2011-01-01
Initial Package Version: 0.6.14
Upstream Status: unknown
Origin: fedora
Description: Grep-2.7 errors out on the 'space' syntax, causing
docbook2html to fail.  Fixed up by hand (the fedora version depends
on a different patch which adds --color=never).

diff -Naur docbook-utils-0.6.14.orig/bin/jw.in docbook-utils-0.6.14//bin/jw.in
--- docbook-utils-0.6.14.orig/bin/jw.in	2003-04-30 17:21:49.000000000 +0100
+++ docbook-utils-0.6.14//bin/jw.in	2011-01-01 18:43:21.558959786 +0000
@@ -80,9 +80,9 @@
 SGML_CATALOGS_DIR="/etc/sgml"
 if [ -f "$SGML_CONF" ]
 then
-  RE='^[:space:]*SGML_BASE_DIR[:space:]*=[:space:]*'
+  RE='^[[:space:]]*SGML_BASE_DIR[[:space:]]*=[[:space:]]*'
   SGML_BASE_DIR=`grep $RE $SGML_CONF | sed "s/$RE//"`
-  RE='^[:space:]*SGML_CATALOGS_DIR[:space:]*=[:space:]*'
+  RE='^[[:space:]]*SGML_CATALOGS_DIR[[:space:]]*=[[:space:]]*'
   SGML_CATALOGS_DIR=`grep $RE $SGML_CONF | sed "s/$RE//"`
 fi
 
@@ -312,7 +312,7 @@
 	  SGML_CATALOG_FILES=$SGML_CENTRALIZED_CATALOG
 	else
 	  SGML_CATALOG_FILES=`find $SGML_BASE_DIR -name catalog`
-	  SGML_CATALOG_FILES=`echo "$SGML_CATALOG_FILES" | tr [:space:] :`
+	  SGML_CATALOG_FILES=`echo "$SGML_CATALOG_FILES" | tr [[:space:]] :`
 	fi
 	;;
   no)	SGML_CATALOG_FILES=""
